org-formation/organization.yaml

```diff
@@ -563,6 +564,22 @@ Organization:
         <<: !Include ./_default_org_tags.yaml
         <<: !Include ./_platform_org_tags.yaml
 
+  RecoverProdAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: org-sagebase-recover-prod
+      AccountId: '634761300905'
+      RootEmail: aws-recover-prod@sagebase.org
```

Comment:
I don't see this alias in Google Groups.  Please see the note in step 2 here: https://sagebionetworks.jira.com/wiki/spaces/IT/pages/2058878986/Bootstrapping+AWS+Project+Accounts

Reply:
Done

Reply:
"Who can post" must be "Anyone on web", lest AWS cannot send email to the endpoint: https://groups.google.com/a/sagebase.org/g/aws-recover-prod/settings?hl=en

Reply:
Done!